File: app/src/main/java/com/d2956987215/mow/activity/mine/WebviewActivity.java
package com.d2956987215.mow.activity.mine;

import android.os.Bundle;
import android.webkit.WebView;

import com.d2956987215.mow.R;
import com.d2956987215.mow.activity.BaseActivity;
import com.d2956987215.mow.rxjava.RxJavaUtil;
import com.d2956987215.mow.util.WebViewUtils;

import butterknife.BindView;

public class WebviewActivity extends BaseActivity implements WebViewUtils.WebViewInterface {

    @BindView(R.id.web)
    WebView web;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        initWeb();
    }

    private void initWeb() {
        new WebViewUtils(web, WebviewActivity.this,this);
        web.loadUrl(RxJavaUtil.HOST + "index.php?m=Api&c=Index&a=aboutUs");
    }

    @Override
    protected String title() {
        return getString(R.string.about_us);
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_webview;
    }

    @Override
    public void show(Object obj) {

    }

    @Override
    public boolean check(WebView webView, String url) {
        return false;
    }

    @Override
    public void JavaScriptObjectMethod(String url) {

    }
}

File: app/src/main/java/com/d2956987215/mow/activity/login/WebViewActivity.kt
package com.d2956987215.mow.activity.login

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.webkit.WebView
import com.d2956987215.mow.R
import com.d2956987215.mow.activity.BaseActivity
import com.d2956987215.mow.util.WebViewUtils
import kotlinx.android.synthetic.main.activity_web_view.*
import kotlinx.android.synthetic.main.include_header.*

class WebViewActivity : BaseActivity(), WebViewUtils.WebViewInterface {
    override fun check(webView: WebView?, url: String?): Boolean {
        return true
    }


    override fun JavaScriptObjectMethod(url: String?) {

    }

    private var title = ""
    private var url = ""
    override fun show(obj: Any?) {

    }

    override fun getLayoutId(): Int {
        return R.layout.activity_web_view
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_web_view)
        title = intent.extras.get("title").toString()
        url = intent.extras.get("url").toString()
        tv_title.text = title
        WebViewUtils(wv, this, this)
        wv.loadUrl(url)
        rl_back.setOnClickListener { finish() }
    }
}
